# Id del robot che sta inviando il messaggio
uint8 robot_id

# Id del waypoint di cui ho calcolato la distanza
uint8 waypoint_id

# Distanza tra robot_id e waypoint_id espressa in [m]
float32 distance